### DISCUSSION TITLE
Will "Hide users posts from feed" stop the scrapper from being able to scrape them?

### DISCUSSION THREAD

**Discussion by RonnieBlaze** [UltimaHoarder/UltimaScraper#775] [Q&A] [answered]
If i hide a users posts will it still scrape?

**Comment by UltimaHoarder** [comment, marked as answer]
It should still scrape, otherwise, you can tell me it doesn't.

**Comment by RonnieBlaze** [comment]
ok i will check it out..  and let you know if it does not work
